FitNesseRoot/DbFit/AcceptanceTests/CoreTests/StoredProcNoParams/content.txt

```diff
@@ -1,5 +1,9 @@
 !3 If there are no parameters, Execute Procedure needs just one row
 
+|Clean|
+|table|clean?|
+|users|true|
```

Comment:
I took a bit more detailed look at core tests - seems like all other tests except `StoredProcNoParams` are already taking care to tear down their changes. So perhaps it's OK to keep the cleanup here in `StoredProcNoParams` but I wold rather vote to move it to the end of the test page (not in the beginning).